network_provisioning/tool/esp_prov/esp_prov.py

```diff
@@ -389,6 +519,12 @@ async def main():
                             'If Wi-Fi scanning is supported by the provisioning service, this need not '
                             'be specified'))
 
+    parser.add_argument('--dataset_tlvs', dest='dataset_tlvs', type=str,
+                        help=desc_format(
+                            'This configures the device to use Dataset Tlvs for the Thread network to '
```

Comment:
This configures the device to use Dataset Tlvs of the Thread network to

Reply:
Done